components/shared/gallery/Properties.vue

```diff
@@ -1,13 +1,25 @@
 <template>
   <CollapseCardWrapper :label="$t('nft.properties.label')">
-    <b-table :data="displayAttributes" :columns="columns" />
+    <div class="columns is-multiline">
+      <div
+        v-for="attribute in attributes"
+        :key="attribute.trait_type"
```

Comment:
```suggestion
        :key="attribute[fieldKey]"
```

Hey, there is a props value `fieldKey`

Reply:
Nice catch!
`<Properties :attributes="meta.attributes" field-key="trait_type" />` 

but why it has been introduced? won't this always be `trait_type`?

Reply:
fixed in new commit